package.json

```diff
@@ -68,9 +68,11 @@
   },
   "dependencies": {
     "choo": "6.6.0",
+    "ipfs": "^0.26.0",
```

Comment:
Let's try to keep all versions locked, so that we don't have big divergence between `npm` and `yarn`, at least on the direct dependency level.

Reply:
Yeah, for IPFS land, always use `~` at least. A minor version might have a breaking change (0.27 will)